website/docs/onboarding/containers/installing-podman.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 10
+title: Installing Podman
+description: Podman Desktop can assist you to install Podman on Windows and macOS.
+tags: [podman-desktop, podman, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos]
+---
+
+# Installing Podman with Podman Desktop
+
+On Windows and macOS, running the Podman container engine requires running a Linux distribution on a virtual machine.
+Podman Desktop can assist you to install the Podman container engine in a Fedora distribution of Linux, on a virtual machine.
+
+Main benefits are:
+
+- Ease of use.
+- On Windows: Windows Subsystem for Linux version 2 (WSL 2) native virtualization performance.
+
+Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if:
+
+- You prefer working with a RHEL distribution of Linux.
```

Comment:
I think the podman-machines are Fedora based though, not RHEL. Maybe ¨RHEL family" but I'm not sure if that makes sense either. 🤔

Reply:
The podman bundlles used by CRC are based on Fedora CoreOS.
For podman on RHEL, starting a MicroShift instance but using it with podman should work.